src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -7024,15 +7030,28 @@ namespace SurfaceGeometry {
             }
         }
 
-        // Change algorithm for Kiva foundation surfaces
+        // Change algorithm for Kiva and air boundary foundation surfaces
         for (auto &surf : Surface) {
             if (surf.ExtBoundCond == KivaFoundation) {
                 surf.HeatTransferAlgorithm = HeatTransferModel_Kiva;
                 DataHeatBalance::AnyKiva = true;
             }
-        }
-
-        // Setup Kiva intances
+            //if (surf.HeatTransSurf) {
+            //    if (DataHeatBalance::Construct(surf.Construction).TypeIsAirBoundaryIRTSurface) {
+            //        // IRT air boundaries use CTF algorithm
+            //        surf.HeatTransferAlgorithm = HeatTransferModel_CTF;
+            //        DataHeatBalance::AnyAirBoundary = true;
+            //    } else if (DataHeatBalance::Construct(surf.Construction).TypeIsAirBoundaryInteriorWindow) {
+            //        surf.HeatTransferAlgorithm = HeatTransferModel_AirBoundaryIntWin;
+            //        DataHeatBalance::AnyAirBoundary = true;
+            //    } else if (DataHeatBalance::Construct(surf.Construction).TypeIsAirBoundary) {
+            //        surf.HeatTransferAlgorithm = HeatTransferModel_AirBoundaryNoHT;
+            //        DataHeatBalance::AnyAirBoundary = true;
+            //    }
+            //}
```

Comment:
Hmmm...not needed?

Reply:
Yeah, I changed approach, but only partially here. `HeatTransferModel_AirBoundary*` are currently only getting used for reporting [here](https://github.com/NREL/EnergyPlus/blob/2323a3a28b12257b00a54039582c8bed4a5230dd/src/EnergyPlus/OutputReports.cc#L1978-L1981). I originally thought these would be used as triggers in the actual heat balance sections, but instead AirBoundaryIRT is just a normal heat transfer surface with IRT properties and AirBoundary_InteriorWindow is being triggerd by looking at something like `Construct(surf.Construction).TypeIsAirBoundaryInteriorWindow`.

Technically, because of the way this decouples the treatment of solar and radiant at the air boundary (a flexible but perhaps regrettable complication) the AirBoundaryInteriorWindow isn't a heat transfer surface - it's just a sunlight/daylight portal with perfect transmittance.

So, I'll need to clean this up later and either remove the `HeatTransferModel_AirBoundary*` variables completely or set them somewhere else.